great-idea/css/index.css

```diff
@@ -62,3 +62,220 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+h1 {
+	font-size: 80px;
+}
+
+.container {
+	width: 880px;
+	margin: 0 auto;
+	border: 3px solid blue;
+  }
+
+header {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	width: 100%;
+	border: 3px solid green;
+}
+
+header nav {
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+	margin-top: 35px;
+	width: 75%;
+	border: 3px solid yellow;
+  }
+
+header nav a {
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+	border: 3px solid red;
+  }
+
+header nav a:hover {
+	text-decoration: underline;
+}
+
+
+.cta {
```

Comment:
Good work making use of inheritance and parent classes. We'll keep coming back to this theme, so it's good that you're getting into this habit early.